static/app/views/performance/newTraceDetails/traceDrawer/traceDrawer.tsx

```diff
@@ -50,11 +50,14 @@ type TraceDrawerProps = {
 };
 
 function getUninitializedDrawerSize(layout: TraceDrawerProps['layout']): number {
-  return layout === 'drawer bottom'
-    ? // 36 of the screen height
-      Math.max(window.innerHeight * 0.36)
-    : // Half the screen minus the ~sidebar width
-      Math.max(window.innerWidth * 0.5 - 220, MIN_TRACE_DRAWER_DIMENSTIONS[0]);
+  const uninitialized =
+    layout === 'drawer bottom'
+      ? // 36 of the screen height
+        Math.max(window.innerHeight * 0.36)
```

Comment:
These values doesn't change when `winner.innerHeight` changes. I believe this is not intended right?

Ref: https://github.com/getsentry/sentry/blob/d5ae1c2c65df7c7431b55fd9439730410392acaa/static/app/views/performance/newTraceDetails/traceDrawer/traceDrawer.tsx#L205